tests/mocks/app.ts

```diff
@@ -46,15 +46,19 @@ export class AppMock {
           () => (this.monaco.latestModel = { updateOptions: jest.fn() }),
         ),
       onDidFocusEditorText: jest.fn(),
-      setTheme: jest.fn(),
       defineTheme: jest.fn(),
+      revealLine: jest.fn(),
+      setTheme: jest.fn(),
     },
     languages: {
       typescript: {
         javascriptDefaults: {
           addExtraLib: jest.fn(),
         },
       },
+      register: jest.fn(),
+      setMonarchTokensProvider: jest.fn(),
+      registerCompletionItemProvider: jest.fn(),
```

Comment:
At some point this custom object is going to be replaced with `new MonacoEditorMock()` so please add these 5 additions into `tests/mocks/monaco-editor` too :bowing_man: